packages/gatsby/package.json

```diff
@@ -178,6 +178,8 @@
     "documentation": "^12.3.0",
     "enhanced-resolve": "^4.2.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
+    "gatsby-transformer-remark": "2.8.46",
+    "gatsby-remark-autolink-headers": "2.3.15",
```

Comment:
Do we have to do this? can't we mock these somehow? Feel free to rip out and do a follow up

Reply:
We currently have to do this for the tests, but I'm not sure we actually even need to add them as we `require.resolve(\`${plugin}/gatsby-node\`)` anyway. Let me test if it still works without them...

Reply:
Works without them, removed! We should definitely mock these at some point, as currently the Gatsby tests depend on the shape of `packages/(gatsby-transfomer-remark|gatsby-remark-autolink-headers)/gatsby-node.js`, but this should be good enough for now 👍